plugins/nf-google/src/main/nextflow/cloud/google/batch/GoogleBatchScriptLauncher.groovy

```diff
@@ -108,24 +109,26 @@ class GoogleBatchScriptLauncher extends BashWrapperBuilder {
     }
 
     List<String> getContainerMounts() {
-        final result = new ArrayList(10)
-        for( String it : pathTrie.longest() ) {
-            result.add("$MOUNT_ROOT$it:$MOUNT_ROOT$it:rw".toString() )
+        return pathTrie.longest().collect { it ->
+            "${MOUNT_ROOT}${it}:${MOUNT_ROOT}${it}:rw".toString()
         }
```

Comment:
`for` is still a better option because prevents the dynamic creation of class required by the closure